src/style.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
```

Comment:
Why only top?

Reply:
I don't know why I use this margin at all. I deleted it.